website/docs/d/oracledatabase_exadata_infrastructure.html.markdown

```diff
@@ -0,0 +1,146 @@
+
+* `max_data_storage_in_tbs` - The total available DATA disk group size.
+
+* `max_db_node_storage_size_in_gbs` - The total local node storage available in GBs.
+
+* `max_memory_in_gbs` - The total memory available in GBs.
+
+* `memory_size_in_gbs` - The memory allocated in GBs.
+
+* `monthly_db_server_version` - The monthly software version of the database servers (dom0) in the cloud Exadata infrastructure. Example: `20.1.15`
+
+* `monthly_storage_server_version` - The monthly software version of the storage servers (cells) in the cloud Exadata infrastructure. Example: `20.1.15`
+
+* `next_maintenance_run_id` - The [OCID](https://docs.oracle.com/en-us/iaas/Content/General/Concepts/identifiers.htm) of the next maintenance run.
+
+* `oci_url` - The URL of the resource in the OCI console.
+
+* `ocid` - The [OCID](https://docs.oracle.com/en-us/iaas/Content/General/Concepts/identifiers.htm) of the Exadata infrastructure.
+
+* `provisioning_state` - CloudExadataInfrastructure provisioning state
+
+* `shape` - The model name of the cloud Exadata infrastructure resource.
+
+* `storage_count` - The number of storage servers for the cloud Exadata infrastructure.
+
+* `storage_server_version` - The software version of the storage servers (cells) in the Exadata infrastructure. Example: `20.1.15`
+
+* `system_data` - A `system_data` block as defined below.
+
+* `tags` - A mapping of tags assigned to the Exadata Infrastructure.
+
+* `time_created` - The date and time the cloud Exadata infrastructure resource was created.
+
+* `total_storage_size_in_gbs` -  The total storage allocated to the cloud Exadata infrastructure resource, in gigabytes (GB).
+
+* `zones` - The Exadata infrastructure Azure zones.
+
+---
+
+A `estimated_patching_time` block exports the following:
+
+* `estimated_db_server_patching_time` - The estimated time required in minutes for database server patching.
+
+* `estimated_network_switches_patching_time` - The estimated time required in minutes for network switch patching.
+
+* `estimated_storage_server_patching_time` - The estimated time required in minutes for storage server patching.
+
+* `total_estimated_patching_time` - The estimated total time required in minutes for all patching operations.
+
+---
+
+A `maintenance_window` block exports the following:
+
+* `days_of_week` - Days during the week when maintenance should be performed.
+
+* `hours_of_day` - The window of hours during the day when maintenance should be performed. The window is a 4 hour slot. Valid values are: 0 - represents time slot 0:00 - 3:59 UTC - 4 - represents time slot 4:00 - 7:59 UTC - 8 - represents time slot 8:00 - 11:59 UTC - 12 - represents time slot 12:00 - 15:59 UTC - 16 - represents time slot 16:00 - 19:59 UTC - 20 - represents time slot 20:00 - 23:59 UTC
+
+* `lead_time_in_weeks` -  Lead time window allows user to set a lead time to prepare for a down time. The lead time is in weeks and valid value is between 1 to 4.
```

Comment:
Since these are the docs for the data source we don't need to specify valid values since users cannot input them here anyway 
```suggestion
* `lead_time_in_weeks` -  Lead time window allows user to set a lead time to prepare for a down time.
```

Reply:
Removed valid value examples.